dbt_coverage/__init__.py

```diff
@@ -353,7 +353,7 @@ def from_column(cls, column: Column, cov_type: CoverageType):
 
         return CoverageReport(cls.EntityType.COLUMN, cov_type, column.name, covered, total, {})
 
-    def to_markdown_table(self):
+    def to_markdown_table(self, cov_type: CoverageType):
```

Comment:
This should not be needed since we can read this from `self.cov_type`. 😊 Can you amend this in all places please?

Reply:
Done! b9a8121ff48966c59270c350c6bb5c3df96b7110